src/foam/nanos/auth/email/EmailTokenService.js

```diff
@@ -35,6 +35,7 @@ foam.CLASS({
 DAO tokenDAO = (DAO) getX().get("localTokenDAO");
 DAO userDAO  = (DAO) getX().get("localUserDAO");
 AppConfig appConfig = user.findGroup(x).getAppConfig(x);
+String url = appConfig.getUrl().replaceAll("/$", "");
```

Comment:
Why not do this in Group.getAppConfig?

Reply:
Sure, we can but will still have to do this for places that are not getting appConfig from group.

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/4616/commits/2c113592cc59709117d65160c87c8605b1415280.